models/app.models.js

```diff
@@ -0,0 +1,7 @@
+const fs = require("fs/promises");
+
+exports.getEndpointsFromFile = () => {
+  return fs.readFile(`${__dirname}/../endpoints.json`).then((contents) => {
```

Comment:
Do we _need_ to read the file here or could we save ourselves the trouble and require in the contents of the file and send that off?